include/global.h

```diff
@@ -117,6 +117,10 @@
 #define DEX_FLAGS_NO (ROUND_BITS_TO_BYTES(NUM_SPECIES))
 #define NUM_FLAG_BYTES (ROUND_BITS_TO_BYTES(FLAGS_COUNT))
 
+// Branch defines:
+// Used by other branches to detect each other.
+#define ITEM_EXPANSION
```

Comment:
move this above the definition of `ROUND_BITS_TO_BYTES` to align with #1213 and #1214

Reply:
Ready